config/routes.rb

```diff
@@ -2,6 +2,11 @@
   crud = %i(index show create update destroy)
 
   namespace :v1 do
-    resources :projects, only: crud
+    resources :projects, only: crud do
+      resources :tasks, only: [:create, :index]
+      # Redirect more RESTful nested requests to /projects/:project_id/tasks/:task_id/ to /tasks/:task_id
+      get '/tasks/:id', to: redirect('/v1/tasks/%{id}')
+    end
+    resources :tasks, only: [:show]
```

Comment:
Since routing for tasks is left completely up to the folks working this challenge I like to ask everyone how they landed on the choices they did. The way you have it here with `tasks#index` and `tasks#create` nested and then `tasks#show` unnested with the bonus redirect for those looking for show nested of course works. You could have also chose to do a full nesting of all task routes or could have chosen to leave tasks at the top level with no nesting.

So just some curiosity on my side as to the driving factors in your routing decision.